concent_api/verifier/tasks.py

```diff
@@ -0,0 +1,51 @@
+from enum import Enum
+import logging
+
+from celery import shared_task
+
+from .constants import VerificationResult
+
+
+logger = logging.getLogger(__name__)
+
+
+@shared_task
+def blender_verification_order(
+    subtask_id:             str,
+    source_package_path:    str,  # pylint: disable=unused-argument
+    result_package_path:    str,  # pylint: disable=unused-argument
+    output_format:          Enum,  # pylint: disable=unused-argument
+    scene_file:             str,  # pylint: disable=unused-argument
+):
```

Comment:
Maybe some logging would be nice here?

Reply:
Off course, but let me at them in the next pull requests, which will put some more code into this task.